validity/src/db/client.rs

```diff
@@ -636,7 +638,9 @@ impl DriverDBClient {
                     .push_bind(req.total_tx_fees.clone())
                     .push_bind(req.l1_chain_id)
                     .push_bind(req.l2_chain_id)
-                    .push_bind(req.contract_address.as_ref().map(|arr| &arr[..]));
+                    .push_bind(req.contract_address.as_ref().map(|arr| &arr[..]))
+                    .push_bind(req.prover_address.as_ref().map(|arr| &arr[..]))
```

Comment:
Sneaked in a missing line here.

Reply:
Do you think there's a more type-safe way to do this to avoid this issue in the future?

Reply:
Hmm.. I don't think there is.